src/groundwater/aquifer.jl

```diff
@@ -107,22 +115,68 @@ aquifer will yield when all water drains and the pore volume is filled by air
 instead. Specific yield will vary roughly between 0.05 (clay) and 0.45 (peat)
 (Johnson, 1967).
 """
-@get_units @grid_loc struct UnconfinedAquifer{T} <: Aquifer
-    head::Vector{T} | "m"  # hydraulic head [m]
+@get_units @grid_loc @with_kw struct UnconfinedAquiferParameters{T}
     k::Vector{T} | "m d-1"  # reference horizontal conductivity [m d⁻¹]
     top::Vector{T} | "m" # top of groundwater layer [m]
     bottom::Vector{T} | "m" # bottom of groundwater layer
     area::Vector{T} | "m2"
     specific_yield::Vector{T} | "m m-1" # [m m⁻¹]
-    conductance::Vector{T} | "m2 d-1" #
-    volume::Vector{T} | "m3" # total volume of water that can be released
     f::Vector{T} | "-" # factor controlling the reduction of reference horizontal conductivity
     # Unconfined aquifer conductance is computed with degree of saturation (only when
     # conductivity_profile is set to "exponential")
 end
 
-storativity(A::UnconfinedAquifer) = A.specific_yield
-storativity(A::ConfinedAquifer) = A.storativity
+function UnconfinedAquiferParameters(dataset, config, indices, top, bottom, area)
+    k = ncread(
+        dataset,
+        config,
+        "lateral.subsurface.conductivity";
```

Comment:
Yes, this will indeed change when we will move to using standard names. 

For consistency it would have been better to change this mapping, but this time I skipped these as it is also not really required for the input parameters (on the other hand, for output it is required). For the SBM refactoring I also changed these mainly to show what is required (for the output) when staying with one-one mapping of parameters and variables.